libs/amount/amount.hpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef IROHA_AMOUNT_H
+#define IROHA_AMOUNT_H
+
+#include <boost/multiprecision/cpp_dec_float.hpp>
+#include <boost/multiprecision/cpp_int.hpp>
+#include <cstdint>
+#include <string>
+
+namespace amount {
+
+  /**
+   * Keeps integer and scale values allowing performing math
+   * operations on them
+   */
+  class Amount {
+   public:
+    /**
+     * Creates Amount with integer = 0 and scale = 0
+     */
+    Amount();
+
+    /**
+     * Amount with integer = amount and scale = 0
+     * @param amount integer part
+     */
+    Amount(boost::multiprecision::uint256_t amount);
```

Comment:
it's fine to place using here (inside the namespace)

Reply:
ok